.circleci/config.yml

```
@@ -146,6 +146,11 @@ jobs:
           command: |
             . env/bin/activate
             make -C docs/ doctest
+      - run:
+          name: linkcheck
+          command: |
+            . env/bin/activate
+            make -C docs/ linkcheck
```

How about we allow failures for `linkcheck`?
```suggestion
            make -C docs/ linkcheck || true
```